htdocs/core/boxes/box_services_expired.php

```diff
@@ -104,7 +104,7 @@ function loadBox($max=5)
     				'logo' => 'company',
     				'url' => DOL_URL_ROOT."/comm/card.php?socid=".$objp->socid);
 
-    				$this->info_box_contents[$i][3] = array('td' => 'class="tdoverflow maxwidth100onsmartphone" align="left"',
+    				$this->info_box_contents[$i][3] = array('td' => 'class="maxwidth100onsmartphone" align="left"',
```

Comment:
A better fix would be to use tdoverflowmax100 or tdoverflowmax200 instead of tdoverflow.
Is rendering ok on your side if you do that ?